src/renderer/components/ft-video-player/ft-video-player.js

```diff
@@ -740,194 +740,317 @@ export default Vue.extend({
+    //         break
+    //       case 50:
+    //         // 2 Key
+    //         // Jump to 20% in the video
+    //         event.preventDefault()
+    //         this.changeDurationByPercentage(0.2)
+    //         break
+    //       case 51:
+    //         // 3 Key
+    //         // Jump to 30% in the video
+    //         event.preventDefault()
+    //         this.changeDurationByPercentage(0.3)
+    //         break
+    //       case 52:
+    //         // 4 Key
+    //         // Jump to 40% in the video
+    //         event.preventDefault()
+    //         this.changeDurationByPercentage(0.4)
+    //         break
+    //       case 53:
+    //         // 5 Key
+    //         // Jump to 50% in the video
+    //         event.preventDefault()
+    //         this.changeDurationByPercentage(0.5)
+    //         break
+    //       case 54:
+    //         // 6 Key
+    //         // Jump to 60% in the video
+    //         event.preventDefault()
+    //         this.changeDurationByPercentage(0.6)
+    //         break
+    //       case 55:
+    //         // 7 Key
+    //         // Jump to 70% in the video
+    //         event.preventDefault()
+    //         this.changeDurationByPercentage(0.7)
+    //         break
+    //       case 56:
+    //         // 8 Key
+    //         // Jump to 80% in the video
+    //         event.preventDefault()
+    //         this.changeDurationByPercentage(0.8)
+    //         break
+    //       case 57:
+    //         // 9 Key
+    //         // Jump to 90% in the video
+    //         event.preventDefault()
+    //         this.changeDurationByPercentage(0.9)
+    //         break
+    //       case 48:
+    //         // 0 Key
+    //         // Jump to 0% in the video (The beginning)
+    //         event.preventDefault()
+    //         this.changeDurationByPercentage(0)
+    //         break
+    //       case 188:
+    //         // , Key
+    //         // Return to previous frame
+    //         this.framebyframe(-1)
+    //         break
+    //       case 190:
+    //         // . Key
+    //         // Advance to next frame
+    //         this.framebyframe(1)
+    //         break
+    //       case 68:
+    //        // D Key
+    //        // Toggle Picture in Picture Mode
+    //        if (!this.player.isInPictureInPicture()) {
+    //          this.player.requestPictureInPicture()
+    //        } else if (this.player.isInPictureInPicture()) {
+    //          this.player.exitPictureInPicture()
+    //        }
+    //        break
+    //      case 27:
+    //        // esc Key
+    //        // Exit full window
+    //        event.preventDefault()
+    //        this.exitFullWindow()
+    //        break
+    //      case 83:
+    //        // S Key
+    //        // Toggle Full Window Mode
+    //        this.toggleFullWindow()
+    //        break
+    //      }
+    //   }
+    //    ...mapActions([
+    //      'calculateColorLuminance'
+    //    ])
+    //  }
```

Comment:
I would prefer these lines removed if we don't need them, we can always revert using git if we need to restore them.